File: Python/lt_01_estseq/lt01_estseq_12.py
ano=(int(input("Insira o valor do Ano Atual: ")))
ano-=(int(input("Insira o valor do Ano de Nascimento: ")))
if ano>=0:
    print("A idade é",ano,"\nDaqui a 17 anos a idade será",ano+17)
else:
    print("Não existe idade negativa!")

File: C/lt_01_estseq/lt01_estseq_12.c
// Nome do Programador:
// Descrição:

#include <stdio.h>

int main() {

  int atual, nascimento;
  do {

    printf("Insira o ano do nacimento: ");
    scanf("%i", &nascimento);
    if (nascimento < 0) {
      printf("Data Inválida!\n");
    }

  } while (nascimento < 0);
  do {

    printf("Insira o ano atual: ");
    scanf("%i", &atual);
    if (atual < nascimento) {
      printf("Data Inválida!\n");
    }

  } while (atual < nascimento);

  printf("\nA idade e %i.\nDaqui a 17 anos tera %i de idade.\n",
         (atual - nascimento), ((atual - nascimento) + 17));
  return 0;
}
